<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IniLetterUi</class>
 <widget class="QWidget" name="IniLetterUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ini Letter</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>&lt;qt&gt;
You must now verify the servers public keys.&lt;br&gt;
You can do this by comparing the hash value given below to the one in the letter from your bank.
&lt;/qt&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="GroupBox26">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Ini-Letter</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="Q3TextView" name="iniBrowser" native="true">
        <property name="toolTip">
         <string>The Ini-Letter of the bank server</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="printButton">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer27">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;qt&gt;
Is the hash value above identical to the one in the letter from your bank?
&lt;/qt&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="goodHashButton">
          <property name="toolTip">
           <string>Click yes if the Ini-Letter is identical to the paper letter from your bank.</string>
          </property>
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="badHashButton">
          <property name="toolTip">
           <string>Click No if the Ini-Letter is not identical with the paper letter from your bank (and contact your bank)</string>
          </property>
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextView</class>
   <extends>QWidget</extends>
   <header>q3textview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
